### PR TITLE
Settings - Writing: introduce a setting for Beautiful Math and Shortcode Embeds in Composing

### LINKED ISSUE #32183: Jetpack Settings - Create beautiful Beautiful Math setting
Beautiful Math will no longer be on by default. Because of this, we need to make a setting for it visible in Jetpack and Calypso settings.

> Use the LaTeX markup language to write mathematical equations and formulas

Copy for info popover:

> LaTeX is a powerful markup language for writing complex mathematical equations and formulas.

Details in Automattic/Jetpack#11931
Jetpack implementation in Automattic/Jetpack#11956

### PR DESCRIPTION
**Pull request by eliorivero**
In the upcoming Jetpack release, Beautiful Math–also known as the LaTeX module–and Shortcode Embeds will no longer be enabled by default.
This PR introduces a setting so users can activate them.

Fixes #32183
Part of https://github.com/Automattic/jetpack/issues/11934

#### Changes proposed in this Pull Request

* add toggle for Beautiful Math
* add toggle for Shortcode Embeds
* add negative -16px top margin and 16px margin when a foldable card precedes a regular card or when a regular card precedes a regular card

Captura de Pantalla 2019-04-16 a la(s) 13 41 09

#### Testing instructions

* go to http://calypso.localhost:3000/settings/writing/YOURSITE
* verify that the toggles are there and that they look good
* toggle Beautiful Math **on**, and verify in https://YOURTSITE/wp-admin/admin.php?page=jetpack_modules&module_tag=Writing that Beautiful Math is **on**
* toggle Beautiful Math **off**, and verify in https://YOURTSITE/wp-admin/admin.php?page=jetpack_modules&module_tag=Writing that Beautiful Math is **off**
* toggle Shortcodes **on**, and verify in https://YOURTSITE/wp-admin/admin.php?page=jetpack_modules&module_tag=Writing that Shortcode Embeds is **on**
* toggle Shortcodes **off**, and verify in https://YOURTSITE/wp-admin/admin.php?page=jetpack_modules&module_tag=Writing that Shortcode Embeds is **off**

### CODE REVIEW

File: client/my-sites/site-settings/composing/latex.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+					text={ translate(
+						'LaTeX is a powerful markup language for writing complex mathematical equations and formulas.'
+					) }
+					link="https://jetpack.com/support/beautiful-math/"
+				/>
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="latex"
+					label={ translate(
+						'Use the LaTeX markup language to write mathematical equations and formulas.'
```
Comment (MichaelArestad) [inline client/my-sites/site-settings/composing/latex.jsx]: Remove the period and this is good to go!

Comment (eliorivero) [reply client/my-sites/site-settings/composing/latex.jsx]: Done. I've added here the toggle for Shortcode Embeds since it's right next to this one.

**Comment by MichaelArestad** [comment]
It's perfect.

File: client/my-sites/site-settings/composing/shortcodes.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+			moduleUnavailable,
+			selectedSiteId,
+			translate,
+		} = this.props;
+
+		return (
+			<Card className="composing__card site-settings__card">
+				<QueryJetpackConnection siteId={ selectedSiteId } />
+				<SupportInfo
+					text={ translate( 'Embed media from popular sites when you author your content.' ) }
```
Comment (MichaelArestad) [inline client/my-sites/site-settings/composing/shortcodes.jsx]: Not necessary for this PR (I can make the change), but I'd like to change the tooltip text to:

```
Shortcodes are WordPress-specific markup that let you add media from popular sites. This feature is no longer necessary as the editor now handles media embeds rather gracefully.
```

Comment (eliorivero) [reply client/my-sites/site-settings/composing/shortcodes.jsx]: Updated 👍

**Pull request merged**